src/core/annotation.js

```diff
@@ -598,6 +580,26 @@ class AnnotationBorderStyle {
   }
 }
 
+/**
```

Comment:
This comment block can be removed because it only states where to find it in the spec, which is something we never really do elsewhere unless really required to understand the code, which isn't the case here.